pkgs/applications/window-managers/hikari/default.nix

```diff
@@ -23,6 +23,15 @@ stdenv.mkDerivation {
     sha256 = "0vxwma2r9mb2h0c3dkpvf8dbrc2x2ykhc5bb0vd72sl9pwj4jxmy";
   };
 
+  patches = [
+    # To fix the build with wlroots 0.14.0:
+    (fetchpatch {
+      url = "https://cgit.freebsd.org/ports/plain/x11-wm/hikari/files/patch-wlroots-0.14?id=f2820b6cc2170feef17989c422f2cf46644a5b57";
```

Comment:
The patch is fetched for a third-party, and not upstream directly. What the reasoning behind this? As is, it is not obvious why one should do so.

Reply:
Yes, that isn't ideal but the patch isn't upstream yet and hub.darcs.net doesn't seem to allow fetching "normal" patches.
The content behind the URL should be stable though, cgit.freebsd.org is a trusted source, and Jan Beich writes most of these patches for wlroots changes anyway (and usually submits them upstream).

Reply:
I'm sorry, I don't know who Jan Beich is, or how they relate to Hikari.

IMO, my understanding of this is not a blocker for merging, so go ahead. I haven't followed dev discussions closely lately, so I probably missed stuff that are obvious, sorry about that.

Reply:
No problem, you didn't miss anything and that was a completely valid and meaningful question. Tbh I should've mentioned it in the commit message or via a comment in the source-code because we typically try to avoid fetching patches form other sources (I was just a bit too lazy to properly explain it :o).